`bin/test -s plone.app.folder` already goes wrong locally:

```
Error in test testReverseMigrate (plone.app.folder.tests.test_migration.TestMigrationHelpers)
Traceback (most recent call last):
  File "/Users/mauritsvanrees/buildout/python2.7/parts/opt/lib/python2.7/unittest/case.py", line 331, in run
    testMethod()
  File "/Users/mauritsvanrees/shared-eggs/plone.app.folder-1.1.0-py2.7.egg/plone/app/folder/tests/test_migration.py", line 95, in testReverseMigrate
    self.assertEqual(list(btree.objectValues()), [])
  File "/Users/mauritsvanrees/shared-eggs/plone.folder-1.0.7-py2.7.egg/plone/folder/ordered.py", line 87, in objectValues
    return LazyMap(self._getOb, self.objectIds(spec))
  File "/Users/mauritsvanrees/shared-eggs/plone.folder-1.0.7-py2.7.egg/plone/folder/ordered.py", line 75, in objectIds
    return ordering.idsInOrder()
  File "/Users/mauritsvanrees/shared-eggs/plone.folder-1.0.7-py2.7.egg/plone/folder/default.py", line 152, in idsInOrder
    return list(self._order())
  File "/Users/mauritsvanrees/shared-eggs/plone.folder-1.0.7-py2.7.egg/plone/folder/default.py", line 163, in _order
    return annotations.get(self.ORDER_KEY, [])
  File "/Users/mauritsvanrees/shared-eggs/zope.annotation-3.5.0-py2.7.egg/zope/annotation/attribute.py", line 43, in get
    annotations = getattr(self.obj, '__annotations__', None)
  File "/Users/mauritsvanrees/shared-eggs/Products.BTreeFolder2-2.13.5-py2.7.egg/Products/BTreeFolder2/BTreeFolder2.py", line 240, in __getattr__
    return self._tree[name]
TypeError: 'NoneType' object has no attribute '__getitem__'
```

The answer, if any, should be in this commit:
https://github.com/zopefoundation/Products.BTreeFolder2/commit/6a0594f9e49d9b603dfa16e2d1c09d1afa089784

--- a/sources.cfg
+++ b/sources.cfg
@@ -151,6 +151,7 @@ zope.globalrequest                  = git ${remotes:zope}/zope.globalrequest.git
 # Products
 Products.Archetypes                 = git ${remotes:plone}/Products.Archetypes.git pushurl=${remotes:plone_push}/Products.Archetypes.git branch=master
 Products.ATContentTypes             = git ${remotes:plone}/Products.ATContentTypes.git pushurl=${remotes:plone_push}/Products.ATContentTypes.git branch=master
+Products.BTreeFolder2               = git git@github.com:zopefoundation/Products.BTreeFolder2.git branch=2.13
 Products.CMFCore                    = git ${remotes:zope}/Products.CMFCore.git pushurl=${remotes:zope_push}/Products.CMFCore.git branch=2.2
 Products.CMFDiffTool                = git ${remotes:plone}/Products.CMFDiffTool.git pushurl=${remotes:plone_push}/Products.CMFDiffTool.git branch=master
 Products.CMFDynamicViewFTI          = git ${remotes:plone}/Products.CMFDynamicViewFTI.git pushurl=${remotes:plone_push}/Products.CMFDynamicViewFTI.git branch=master

--- a/versions-blacklist.cfg
+++ b/versions-blacklist.cfg
@@ -1,4 +1,3 @@
-Products.BTreeFolder2 = 2.13.4
 Unidecode = 0.04.2
 Unidecode = 0.04.3
 Unidecode = 0.04.4

--- a/versions.cfg
+++ b/versions.cfg
@@ -10,8 +10,7 @@ docutils = 0.12
 AccessControl = 3.0.11
 # More memory efficient version, Trac #13101
 DateTime = 3.0.3
-# Products.BTreeFolder2 2.13.4 causes a regression
-Products.BTreeFolder2 = 2.13.3
+Products.BTreeFolder2 = 2.13.5
 # Override until ztk is updated
 Sphinx = 1.1.3
 # required for recent z3c.form and chameleon

